tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -0,0 +1,91 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Run kubevirt testing
+# Maintainer: Nan Zhang <nan.zhang@suse.com>
+
+use base multi_machine_job_base;
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    my $ip_out = script_output('ip route show | grep -Eo "src\s+([0-9.]*)\s+" | head -1 | cut -d\' \' -f 2', 30);
+
+    set_var('AGENT_IP', $ip_out);
+    bmwqemu::save_vars();
+
+    # Mark ready state
+    mutex_create('DST_READY_TO_START');
+
+    my $server_ip = $self->get_var_from_child("SERVER_IP");
+
+    rke2_agent_setup($self, $server_ip);
+}    
+# Mutex lock/unlock functionality
+sub set_mutex_lock {
+    my ($self, $lock_signal) = @_;
+
+    mutex_lock($lock_signal);
+    mutex_unlock($lock_signal);
+}
+
+sub rke2_agent_setup {
+    my ($self, $server_ip) = @_;
+
+    systemctl('stop apparmor.service');
+    # systemctl('stop SuSEfirewall2.service');
+    assert_script_run('ssh-keygen -b 2048 -t rsa -q -N "" -f ~/.ssh/id_rsa') unless (! -e "~/.ssh/id_rsa");
+    exec_and_insert_password("ssh-copy-id -o StrictHostKeyChecking=no -i /root/.ssh/id_rsa.pub root\@$server_ip");
+
+    # Ensure SUSE certificates are installed on the node
+    ensure_ca_certificates_suse_installed();
+
+    # Install downstream kubevirt packages
+    zypper_call('in -n kubernetes1.18-client') if (script_run('rpmquery kubernetes1.18-client'));
```

Comment:
I think capability of installing kubexxxx package with customizable version is better.

Reply:
Fixed. Referring to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14364#discussion_r817562025

Reply:
Fixed but not committed here ?

Reply:
Is it commited or not? @nanzhg

Reply:
I revert it back because there is no choice for this package.

```
# zypper se kubernetes
Refreshing service 'Basesystem_Module_15_SP4_x86_64'.
Refreshing service 'Containers_Module_15_SP4_x86_64'.
Refreshing service 'Desktop_Applications_Module_15_SP4_x86_64'.
Refreshing service 'Development_Tools_Module_15_SP4_x86_64'.
Refreshing service 'Legacy_Module_15_SP4_x86_64'.
Refreshing service 'SUSE_Linux_Enterprise_Server_15_SP4_x86_64'.
Refreshing service 'Server_Applications_Module_15_SP4_x86_64'.
Loading repository data...
Reading installed packages...

S  | Name                         | Summary                              | Type
---+------------------------------+--------------------------------------+--------
i+ | kubernetes1.18-client        | Kubernetes client tools              | package
i  | kubernetes1.18-client-common | Kubernetes client tools common files | package
```